pom.xml

```diff
@@ -39,6 +26,14 @@
             <url>http://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
+    <dependencies>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-aggregator</artifactId>
+            <version>1.4.2</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
 </project>  
```

Comment:
@recena I've seen that and it's not right or wrong (hence the fact that I never commented on it), but I personally think using `workflow-aggregator` is better for `test` scope simply because it's easier to maintain the dependency. For non `test` scope, one could argue that it should be more explicit.